tests/foreman/ui/test_host.py

```diff
@@ -2143,6 +2143,127 @@ def test_bootc_booted_container_images(target_sat, bootc_host, function_ak_with_
         assert booted_container_image_info[1]['Hosts'] == '1'
 
 
+@pytest.mark.tier2
+def test_bootc_host_details(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Create a bootc host, and read it's information via the Host Details UI
+
+    :id: 842356e9-8798-421d-aca6-0a1774c3f22b
+
+    :expectedresults: Host Details UI contains the proper information for a bootc host
+
+    :CaseComponent:Hosts-Content
+
+    :Verifies:SAT-27171
```

Comment:
No functional impact, just for consistency.
```suggestion
    :CaseComponent: Hosts-Content

    :Verifies: SAT-27171
```